### DISCUSSION TITLE
Create Bevy Matrix server

### DISCUSSION THREAD

**Discussion by ghost** [bevyengine/bevy#3661] [Ideas]
Nowadays, many open-source communities have a decentralised Matrix server for their community. Maybe Bevy should become one of them...?

**Comment by bjorn3** [comment]
There is an unofficial bevy room at [#bevyengine:matrix.org](https://matrix.to/#/!QTfdWupQFAIrzmqIAB:matrix.org). If there were to be an official bevy room or space on matrix I would personally like it to be bridged with the discord so people can choose either without missing out on anything.

Also I will convert this to a discussion as this isn't a bug in bevy.

**Reply by blueforesticarus** [reply]
~~This unfortunately appears to be entirely dead.~~

EDIT: Okay, there apparently *are* some semi-active people there.

**Comment by K4LCIFER** [comment]
A Matrix server bridged to Discord, at the very least, if not moving completely to Matrix, would be a wise move for a myriad of reasons.
